### PR TITLE
Remove more wstrings

### PR DESCRIPTION
**Pull request by dimitry-**
This PR removes more wstrings and moves all code over to absl::StartsWith/EndsWith/StrContains

### CODE REVIEW

File: OrbitQt/outputdialog.cpp [hunk 1]
```diff
@@ -23,7 +23,7 @@ void OutputDialog::SetStatus(const std::string& a_Status) {
...
 
 //-----------------------------------------------------------------------------
-void OutputDialog::AddLog(const std::wstring& a_Log) {
-  std::wstring log = ui->OutputTextEdit->toPlainText().toStdWString() + a_Log;
-  ui->OutputTextEdit->setPlainText(QString::fromWCharArray(log.c_str()));
-}
\ No newline at end of file
+void OutputDialog::AddLog(const std::string& a_Log) {
+  std::string log = ui->OutputTextEdit->toPlainText().toStdString() + a_Log;
+  ui->OutputTextEdit->setPlainText(log.c_str());
```
Comment (beckerhe) [inline OrbitQt/outputdialog.cpp:28]: `QString::fromStdString` would be better here.

Comment (beckerhe) [reply OrbitQt/outputdialog.cpp:28]: Or maybe even convert a_Log to a QString and make the concatenation in QString-land. I'm not really sure which is nicer/faster/cleaner.

Comment (dimitry-) [reply OrbitQt/outputdialog.cpp:28]: setPlainText expects QString?

Comment (dimitry-) [reply OrbitQt/outputdialog.cpp:28]: done

**Comment by beckerhe** [comment]
LGTM, but the windows build has a different opinion on that.

**Comment by dimitry-** [comment]
Added commit fixing windows build.

File: external/DIA2Dump/dia2dump.cpp [hunk 2]
```diff
@@ -10,7 +10,7 @@
 // Debug Interface Access SDK API.
 //
 
-#include "Dia2Dump.h"
+#include "dia2dump.h"
```
Comment (beckerhe) [inline external/DIA2Dump/dia2dump.cpp:13]: This was necessary to make it compile for windows?

Comment (dimitry-) [reply external/DIA2Dump/dia2dump.cpp:13]: not really :) I was trying to bring consistency to includes..

**Pull request merged**